src/encoded/tests/test_access_key.py

```diff
@@ -160,4 +160,4 @@ def test_access_key_uses_edw_hash(app, access_key):
     root = app.registry[COLLECTIONS]
     obj = root.by_item_type['access_key'][access_key['access_key_id']]
     pwhash = obj.properties['secret_access_key_hash']
-    assert EDWHash.encrypt(access_key['secret_access_key']) == pwhash
```

Comment:
oh this must get rid of one of the annoying dependencies.

Reply:
Yea, cryptography was removed.  There were a few changes for that in lrowe's pr.